internal/tools/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/tcnksm/ghr v0.16.0
 	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad
 	go.opentelemetry.io/build-tools/checkdoc v0.0.0-20220919170044-dd98e8af322a
+	go.opentelemetry.io/build-tools/chloggen v0.0.0-20220928154055-b00fb8eb9066
```

Comment:
Should the versions for tools in `go.opentelemetry.io/build-tools` be aligned?

Reply:
Fixed for consistency